docs/reference/annotations.rst

```diff
@@ -1,6 +1,12 @@
 Annotations
 ===========
 
+.. note::
+
+    This documentation is outdated and will only work with symfony 2/3.
```

Comment:
In this case we should remove the following docs completely and link to the cookbook recipe we have from marko

Reply:
Removed the chapter, there is a recipe for Markos bundle